### ISSUE TITLE
Uncaught exception Error

### ISSUE THREAD

**Issue by philm001** [airbnb/lottie-web#2954]
I have a simple issue. I am using the edge browser's latest version. Happens in the firefox browser, also latest version. I am using the sample located here:

https://lottiefiles.com/interactivity

Basically, I have a WordPress post that I am adding in a Lottie animation. Inside a custom HTML box, I have the Lottie code as follows:

`
<script src="https://unpkg.com/@lottiefiles/lottie-interactivity@latest/dist/lottie-interactivity.min.js">
    
    
</script>

<lottie-player id="firstLottie" src="https://assets2.lottiefiles.com/packages/lf20_i9mxcD.json" style="width:400px; height: 400px;">"></lottie-player>

<script>
LottieInteractivity.create({
    player: '#firstLottie',
    mode: 'scroll',
    actions: [
        {
            visibility:[0,1.0],
            type: 'seek',
            frames: [0, 300],
        },
    ]
});
</script>
`

Basically, I get a `Uncaught TypeError: W.getLottie is not a function on the line: `LottieInteractivity.create({` I am not sure what is wrong here. Do I have everything in the correct order?

**Comment by bodymovin** [comment]
hi, can you create the issue in the lottiefiles repository?

**Comment by kamalx** [comment]
Hello, I got the same error on trying the code on the said url above by philm001. Any chance this issue was posted elsewhere and resolved? Any links here would be very helpful. Thank you!

**Comment by Jay-Plumb** [comment]
+1 I notice the exact same issue within the demo

**Comment by kartik-solvative** [comment]
+ 1 Facing the same issue

**Comment by danieltpham** [comment]
+1

**Comment by bodymovin** [comment]
this issue should be reported in the lottiefiles repository
